Project Setup:

okikiola11 opened the pull request:
## I worked on the following features:
- Used create-react-app to initialize a new React app for the bookstore project
- Deleted unnecessary files or assets
- Added prop-types using npm for type checking props.
- Added the necessary Redux packages to the project:
-- Added react-redux using npm.
-- Added redux using npm.
- Ran the application with npm start and made sure that everything works correctly.
- Initialized a git repo
- Created a Heroku app
- Committed and deployed to Heroku
- Made sure the Heroku app works correctly
- Pushed everything to Github

okikiola11 committed:
Add the linting configuration
Add eslint, stylelint, babel and github workflows linters
Remove unnecessary files from the application

afanimartin reviewed (approved):
## Approved 👍🏿 👍🏿 👍🏿 
Nice work. Necessary packages added to `package.json` and readme looks professional. Well done!

### Martin Afani
#### Slack: whiz25

okikiola11 committed:
Add the react-redux and redux dependency package
Add the necessary eslint, stylelint dependency package
Add the proptypes package
Fix eslint errors

okikiola11 committed:
Update the README.MD file with the project setup instructions
Create an heroku link for the project

okikiola11 committed:
Add the heroku link to the README.md file

okikiola11 committed:
Remove the app.test.js file
